itests/org.jboss.tools.cdk.ui.bot.test/src/org/jboss/tools/cdk/ui/bot/test/server/wizard/download/MinishiftDownloadRuntimeTest.java

```diff
@@ -36,7 +36,7 @@ public MinishiftDownloadRuntimeTest(CDKVersion version) {
 
 	@Parameters(name = "{0}")
 	public static Collection<CDKVersion> data() {
-		return Arrays.asList(CDKVersion.MINISHIFT1270);
+		return Arrays.asList(CDKVersion.MINISHIFT1310);
```

Comment:
Any reason this is MINISHIFT1310 where this is MINISHIFT1320 elsewhere ?

Reply:
Not really. But I still do not mind testing downloading differenr version as well.